EclipseMarketplaceLicense Test failing

:clock11: **When did the problem start?**
~1 day ago

<!-- Indicate when the problem started -->

:camera: **Live badge**
Badge is fine, test needs to be updated as target project changed license from GPL to EPL-2.0
<!-- Provide a link to the live badge in plain text and markdown.  -->

![]( url goes here )

:wrench: **Is the live badge working?**
Yup

<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://circleci.com/gh/badges/daily-tests/389
<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**

```
AssertionError: message mismatch: expected 'EPL 2.0' to equal 'GPL'
```

:bulb: **Possible solution**
Update the test to reflect the license change in https://github.com/PyvesB/Notepad4e/commit/dd1eab9450b9d0ac6cea6c978e6b6287f33ca9f0
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/eclipse-marketplace/eclipse-marketplace-license.tester.js
+++ b/services/eclipse-marketplace/eclipse-marketplace-license.tester.js
@@ -6,7 +6,7 @@ t.create('license')
   .get('/notepad4e.json')
   .expectBadge({
     label: 'license',
-    message: 'GPL',
+    message: 'EPL 2.0',
   })
 
 t.create('unspecified license')

